utils/rpms/daos.spec

```diff
@@ -592,6 +592,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Mon Sep 23 2024 Kris Jacque <kris.jacque@intel.com> 2.3.103-6
```

Comment:
Looks like the version got goofed up here.

Reply:
Good catch. Fixed.